app/editQuestion.html

```diff
@@ -0,0 +1,86 @@
+    <script defer src="js/sendQuestionForm.js"></script>
+    <title>Edit Question</title>
+</head>
+<body>
+<div class="container">
+    <img class="logo" alt="Mayden Logo" src="images/branding-icons/maydenacademy_logotype_05.18_web.svg">
+    <div class="form">
+        <h3>Edit question</h3>
+        <form id="edit-question">
+            <div class="form-group">
+                <label for="questionText">Question:</label>
+                <textarea class="form-control" id="questionText" rows="5" name="question"></textarea>
+            </div>
+            <div class="answers">
+                <h4 class="page-text">Enter up to 5 answers below (minimum 2) then tick corresponding checkbox to
+                    indicate correct answer before submitting</h4>
+                <div class="form-group">
+                    <label for="option1">Option 1:</label>
+                    <textarea class="form-control answer" id="option1" rows="2" name="option1"></textarea>
+                    <input class="answer-check" type="radio" name="answer-check" data-question-id="1">
+                </div>
+                <div class="form-group">
+                    <label for="option2">Option 2:</label>
+                    <textarea class="form-control answer" id="option2" rows="2" name="option2"></textarea>
+                    <input class="answer-check" type="radio" name="answer-check" data-question-id="2">
+                </div>
+                <div class="form-group">
+                    <label for="option3">Option 3:</label>
+                    <textarea class="form-control answer" id="option3" rows="2" name="option3"></textarea>
+                    <input class="answer-check" type="radio" name="answer-check" data-question-id="3">
+                </div>
+                <div class="form-group">
+                    <label for="option4">Option 4:</label>
+                    <textarea class="form-control answer" id="option4" rows="2" name="option4"></textarea>
+                    <input class="answer-check" type="radio" name="answer-check" data-question-id="4">
+                </div>
+                <div class="form-group">
+                    <label for="option5">Option 5:</label>
+                    <textarea class="form-control answer" id="option5" rows="2" name="option5"></textarea>
+                    <input class="answer-check" type="radio" name="answer-check" data-question-id="5">
+                </div>
+                <div class="form-group">
+                    <label for="test_id">Assign to test:</label>
+                    <select class="custom-select" id="test_id">
+                    </select>
+                </div>
+            </div>
+            <div id="message-target"></div>
+            <div class="buttons">
+                <div class="messageArea">
+                    <span id="inputSubmissionConfirmation" class="alert" role="alert"></span>
+                </div>
+                <div class="buttons">
```

Comment:
these button styles are not consistent with the rest of the application, can you use the same classes/styles as the admin page purple buttons please